lib/coordinates.rb

```diff
@@ -0,0 +1,59 @@
+class Coordinates
+  def initialize(coords)
+    @coords = coords.split(" ")
+  end
+
+  def are_diagonal?
+    not are_vertical? and not are_horizontal?
+  end
+
+  def are_vertical?
+    expected_length = @coords.length
```

Comment:
Suggest removing this line because it does nothing; no output, right?

Reply:
That is a great refactoring task!